src/Controller/Kobo/Api/ApiEndpointController.php

```diff
@@ -16,8 +17,8 @@ public function __construct(
     }
 
     #[Route('/', name: 'api_endpoint')]
-    public function index(): Response
+    public function index(KoboDevice $koboDevice): Response
     {
-        return new Response('<html><body>Hello Kobo</body></html>');
+        return new Response(sprintf('<html><body>%s</body></html>', htmlentities('Hello Kobo "'.($koboDevice->getName() ?? $koboDevice->getId()).'"')));
```

Comment:
Wouldn't it be cleaner to return a jsonresponse instead? that would allow to write a test of the auth + param injection by checking the json fields